service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -271,12 +271,17 @@
     }
 
     private LocalDate dateSubmitted;
+    private final String usePbaPayment;
+    private final String pbaNumber;
+    private final String clientCode;
+    private final String fileReference;
```

Comment:
I'm not big fan of this one, When i look at this variable i have no idea what it represents. The fileReference is really generic name. Ideally could be like 
```
    @JsonUnwrapped
    @Builder.Default
    private final UploadDraftOrdersData uploadDraftOrdersEventData = UploadDraftOrdersData.builder().build();
    
   but i understand that is may require a lot of changes, so i guess can stay